brianlong/www.validators.app
# 187045809 - add zippy stake pool

#### What's this PR do?
- Adds zippy stake pool.

#### How should this be manually tested?
- `rails add_stake_pools:mainnet` & `rails update_fees_in_stake_pools:mainnet`
- enter stake pools page and verify new stake pool. Test the filter form
- check content on main page. There are list of stake pools too. Verify logo and zippy stake link.

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/187045809)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
